spec/models/school_spec.rb

```diff
@@ -36,6 +36,13 @@
 
   context "delegations" do
     it { should delegate_method(:name).to(:gias_school) }
+    it { should delegate_method(:postcode).to(:gias_school) }
+    it { should delegate_method(:town).to(:gias_school) }
+    it { should delegate_method(:telephone).to(:gias_school) }
+    it { should delegate_method(:website).to(:gias_school) }
+    it { should delegate_method(:address1).to(:gias_school) }
+    it { should delegate_method(:address2).to(:gias_school) }
+    it { should delegate_method(:address3).to(:gias_school) }
```

Comment:
If we use a presenter for the address fields, we probably don't need to delegate postcode, town, address1, 2, 3.

Reply:
I'd quite like to keep the delegations in, as I think there is a benefit to being about to use `school.postcode` and not need to require the `GiasSchool` association.